.cspell.json

```diff
@@ -0,0 +1,133 @@
+{
+    "version": "0.2",
+    "language": "en_US",
+    "words": [
+        "ABAC",
+        "AWSCLIV",
+        "Armon",
+        "Brolliar",
+        "CNCF",
+        "Codecov",
+        "Docsify",
+        "Docu",
+        "ETAG",
+        "Ecdnendpoint",
+        "Ecdnprofile",
+        "Errorf",
+        "Fioravanti's",
+        "Fontaine",
+        "GWLB",
+        "HTTPIP",
+        "Inclusivity",
+        "Intelli",
+        "Intn",
+        "Katacode",
+        "Kial",
+        "LBBE",
+        "Liatrians",
+        "Liatrio",
+        "Pactionly",
+        "Phippy",
+        "Qube",
+        "SCTP",
+        "SDLC",
+        "Schreiber",
+        "Servlet",
+        "Souza",
+        "Uncompress",
+        "Unsets",
+        "VMSS",
+        "VNET",
+        "acrpull",
+        "adoptopenjdk",
+        "aluable",
+        "azurerm",
+        "blaa",
+        "boto",
+        "burnup",
+        "cfssl",
+        "cfssljson",
+        "cifs",
+        "dbservers",
+        "devel",
+        "devopsbootcamp",
+        "distro",
+        "distros",
+        "dōjō",
+        "eddieb",
+        "egotiable",
+        "elete",
+        "fluentd",
+        "gesparza",
+        "gofmt",
+        "healthz",
+        "hpas",
+        "joda",
+        "jodatime",
+        "kdeardorff",
+        "keepassxc",
+        "kubebuilder",
+        "kubelet",
+        "kubelets",
+        "kungfu",
+        "liatr",
+        "msdemo",
+        "mydeployment",
+        "myfile",
+        "mypod",
+        "myservice",
+        "ndependent",
+        "nqzxz",
+        "octumn",
+        "pdate",
+        "pvcs",
+        "reate",
+        "schedulable",
+        "shuf",
+        "stimable",
+        "struct",
+        "structs",
+        "terraformrc",
+        "testvm",
+        "tfsdk",
+        "tfvars",
+        "theia",
+        "uncontainerized",
+        "uncordon",
+        "unstaged",
+        "westus",
+        "zxrnr"
```

Comment:
I assume these are words we ignore. And I trust your judgement that they should be. But at a glance I am thinking "what?! how are `zxrnr` or `nqzxz` words"

Reply:
correct, this tells `cspell` that these words defined here are all good.

Reply:
more info:
`words` - a list of words to be considered correct.
`ignoreWords` - a list of words to be ignored (even if they are in the flagWords).